codec/std/compact_bit_array.go

```diff
@@ -0,0 +1,222 @@
+package std
```

Comment:
This shouldn't go in `std`. Just the proto `oneof`s themselves.

Also where are the tests for this?